ui/src/common/components/CitationsByYearGraph/__tests__/CitationsByYearGraph.test.jsx

```diff
@@ -94,4 +94,16 @@ describe('CitationsByYearGraph', () => {
     wrapper.update();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('renders filling missing years with 0', () => {
+    const citationsByYear = {
+      '2000': 10,
+      '2014': 43,
+      '2015': 5,
+    };
+    const wrapper = shallow(
+      <CitationsByYearGraph citationsByYear={citationsByYear} />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
what if ``citationsByYear`` is ``{}``?

Reply:
There is already a test for that https://github.com/inspirehep/inspirehep/blob/caad249aa4251a61f2f6d9a936a52a6256d92b1b/ui/src/common/components/CitationsByYearGraph/__tests__/CitationsByYearGraph.test.jsx#L40